testchain/node/Cargo.toml

```diff
@@ -12,17 +12,17 @@ targets = ["x86_64-unknown-linux-gnu"]
 name = "spacewalk-standalone"
 
 [build-dependencies]
-substrate-build-script-utils = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0" }
+substrate-build-script-utils = { workspace = true }
 
 [dependencies]
-clap = { version = "4.0.26", features = ["derive"] }
-codec = { package = "parity-scale-codec", version = "3.0.0" }
-hex-literal = "0.2.1"
-log = "0.4.8"
-serde = { version = "1.0.130", features = ["derive"] }
-serde_json = "1.0.85"
-futures = "0.3.15"
-tokio = { version = "1.37", features = ["full", "tracing"] }
+clap = { version = "4.5.3", features = [ "derive" ] }
```

Comment:
Node requires a higher version of `clap` compared to that used in the workspace.